autoscheduler/frontend/src/components/SchedulingPage/ConfigureCard/ConfigureCard.tsx

```diff
@@ -144,18 +69,18 @@ const ConfigureCard: React.FC = () => {
             ? <SmallFastProgress />
             : 'Generate Schedules'}
         </Button>
-      </div>
-      <Snackbar
-        open={showSnackbar}
-        autoHideDuration={5000}
-        message="No schedules found. Try widening your criteria."
-        onClose={handleSnackbarClose}
-        action={(
-          <IconButton aria-label="close" onClick={(): void => setShowSnackbar(false)}>
-            <CloseIcon fontSize="small" style={{ color: 'white' }} />
-          </IconButton>
+        <Snackbar
+          open={!!snackbarMessage}
```

Comment:
Really minor but just for future reference it's more "typescripty" (that's totally a word) to use `Boolean(x)` instead of `!!x` since it's more clear that you're typecasting, I'll just make this change in the generate schedules error message PR though since it's really small, I'll be changing this part of the code anyways, and I don't want to hold up this PR any more